apps/src/blockly/eventHandlers.ts

```diff
@@ -53,13 +58,15 @@ export function disableOrphans(event: GoogleBlockly.Events.Abstract) {
   }
 
   const eventWorkspace = Blockly.Workspace.getById(blockEvent.workspaceId);
-  const block = eventWorkspace?.getBlockById(blockEvent.blockId);
+  const block = eventWorkspace?.getBlockById(
+    blockEvent.blockId
+  ) as ExtendedBlock;
   if (
     blockEvent.type === Blockly.Events.BLOCK_MOVE ||
     blockEvent.type === Blockly.Events.BLOCK_CREATE ||
     isEnabledEvent
   ) {
-    if (block) {
+    if (block && !block.getFillPattern()) {
```

Comment:
This change is later reverted in favor of never adding add this event handler to a Jigsaw workspace in the first place.